lib/exercism/user_finished_tracks.rb

```diff
@@ -11,6 +11,23 @@ def initialize(user)
   end
 
   def completed_tracks
+    advanced_tracks = {}
+    started_tracks.group_by{|e| e["track_id"]}
```

Comment:
Hey @MohanL, thank you for the PR! 🎉 

Since we are still using `group_by` here we could let `Postgres` do the job for us and even make it return the problems per track, so we would receive something like this:

```ruby
[{
  "track_id"=>"lua",
  "completed_problems"=>"bob,rna-transcription,food-chain,anagram,phone-number,grade-school,hamming,nucleotide-count,beer-song,word-count,robot-name"
 },
 {
   "track_id"=>"rust",
   "completed_problems"=>"hamming,difference-of-squares,gigasecond,beer-song,scrabble-score,raindrops,leap,hello-world,bob"
}]
```